src/main/java/uk/gov/ons/census/casesvc/service/CaseReceiptService.java

```diff
@@ -43,18 +43,29 @@ private void setUpRules() {
     */
 
     rules.put(new Key("HH", "U", HH), receiptAndCancel);
+    rules.put(new Key("HH", "U", IND), receiptAndCancel);
```

Comment:
Since you've found this scenario is actually impossible, would it be best to remove the rule entirely?